<launch>

  <!-- start PointCloud Fast Filter -->
  <node name="astra_ped" pkg="pff_ped" type="astra_ped" output="screen">
    <param name="robot_height" value="6.0"/> <!-- Default 0.5 (50cm) -->
    <param name="sensor_height" value="0.54"/> <!-- Default 0.15 (15cm) -->
    <param name="horizontal_fov" value="360"/> <!-- Default 360° -->
    <param name="resolution" value="0.4"/> <!-- Default 0.4° -->
    <param name="legs_begin" value="0.25"/> <!-- Default 0.25 (50cm) -->
    <param name="legs_end" value="0.60"/> <!-- Default 0.60 (60cm) -->
    <param name="trunk_begin" value="0.80"/> <!-- Default 0.80 (80cm) -->
    <param name="trunk_end" value="1.50"/> <!-- Default 1.50 (150cm) -->
    <param name="topic_pub_people" value="/People_PC"/> <!-- Default /People_PC -->
    <param name="topic_pub_legs" value="/Legs_PC"/> <!-- Default /Legs_PC -->
    <param name="topic_pub_trunk" value="/Trunk_PC"/> <!-- Default /Trunk_PC -->
    <param name="topic_sub" value="/camera/depth/points"/> <!-- Default velodyne_points -->
    <param name="frame_id" value="camera_depth_frame"/> <!-- Default base_scan -->
  </node>

  <!-- start rviz -->
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find pff_ped)/rviz_cfg/astra_people.rviz" />

</launch>
